README.md

```diff
@@ -179,22 +179,14 @@ sudo pacman -S qt5-base \
 sudo emerge -a ">=dev-qt/qtgui-5.9.0" media-libs/fontconfig
 ```
 
-##### Ubuntu 16.04
-
-```bash
-sudo add-apt-repository ppa:beineri/opt-qt592-xenial
-sudo add-apt-repository ppa:george-edison55/cmake-3.x
-sudo add-apt-repository ppa:ubuntu-toolchain-r-test
-sudo apt-get update
-sudo apt-get install -y g++-7 qt59base qt59svg qt59tools qt59multimedia cmake liblmdb-dev libsodium-dev
-```
-
-##### Ubuntu 19.10
+##### Ubuntu 20.04
 
 ```bash
 # Build requirements + qml modules needed at runtime (you may not need all of them, but the following seem to work according to reports):
-sudo apt install g++-7 cmake liblmdb-dev libsodium-dev libssl-dev qt{base,declarative,tools,multimedia,script,quickcontrols2-}5-dev qml-module-qt{gstreamer,multimedia,quick-extras,-labs-settings,graphicaleffects,quick-controls2} libqt5svg5-dev
```

Comment:
Did you intentionally remove the qml modules? Those are needed at runtime. So while you can build it without them, you will get a blank window! :D

Reply:
You are right, the chat window remains empty without them. I wonder why nheko from Ubuntu repos works just fine [without any qml module installed](https://packages.ubuntu.com/focal/nheko).